client/components/mma/accountoverview/ManageProduct.tsx

```diff
@@ -156,6 +157,7 @@ const InnerContent = ({
 			)}
 
 			{(isAmountOveridable || showSupporterPlusUpdateAmount) &&
+			!isNonServiceableCountry &&
```

Comment:
you're checking the truthiness of the function imported, not calling the function on the productDetail